compiler-plugin/src/main/java/io/ballerina/stdlib/persist/compiler/DiagnosticsCodes.java

```diff
@@ -73,11 +73,14 @@ public enum DiagnosticsCodes {
             "references for relation", ERROR),
     PERSIST_123("PERSIST_123", "associated entity does not contain any primary keys: the key " +
             "should have a valid value", ERROR),
-    PERSIST_124("PERSIST_124", "The entity ''{0}'' should be a closed record.", ERROR),
+    PERSIST_124("PERSIST_124", "the entity ''{0}'' should be a closed record.", ERROR),
```

Comment:
Lets follow the same convention in all codes. Use a simple case to start.

Reply:
Checked.